Source/WebKit/UIProcess/WebProcessProxy.h

```diff
@@ -183,7 +183,8 @@ class WebProcessProxy final : public AuxiliaryProcessProxy {
     void removeSuspendedPageProxy(SuspendedPageProxy&);
 
     WebProcessPool* processPoolIfExists() const;
-    WebProcessPool& processPool() const;
+    // This function is implemented in WebProcessPool.h.
```

Comment:
The should be at the end of the next line. Otherwise, it won't show in the compilation error if someone is missing the include.

Reply:
Fixed!